cli/module_generate/templates/python/tmpl-requirements.txt

```diff
@@ -0,0 +1,6 @@
+{{ if eq .ResourceSubtype "mlmodel" }}
+viam-sdk[mlmodel]
```

Comment:
@purplenicole730 we still want to pin this to `{{ .SDKVersion }}`

Reply:
Thanks for catching that! I'll fix that in my next PR